components/event-feed-service/pkg/persistence/mappings.go

```diff
@@ -159,12 +158,10 @@ var feedProps = `
 // Feeds mapping used to create the feeds-[version] index
 var Feeds = Mapping{
 	Index:      IndexNameFeeds,
-	Type:       DocType,
 	Timeseries: false,
 	Properties: fmt.Sprintf(`{ %s }`, feedProps),
 	Mapping: fmt.Sprintf(`
 			{
-				"template":"`+IndexNameFeeds+`",
```

Comment:
In ES 7.X Elasticsearch changed from 5 shards to 1 shard as the default. This doesn't effect standalone, but could cause some issues on clustered deploys.

To get the previous behavior, we'll need to add "shards": 5, to all of these templates.

Reply:
Thank you, I will add that.